core/domain/user_services_test.py

```diff
@@ -2195,9 +2200,11 @@ def test_cannot_update_dashboard_stats_log_with_invalid_schema_version(
     def test_flush_migration_bot_contributions_model(self):
```

Comment:
Do we still need the `flush_migration_bot_contributions` method? If no, should we remove this in the next release? (After flushing the data?)

Reply:
Makes sense - removed (We have already deleted the model).